tests/test_core.py

```diff
@@ -11,7 +11,7 @@
 from geopandas.testing import assert_geodataframe_equal, assert_geoseries_equal
 
 
-@pytest.fixture
+@pytest.fixture()
```

Comment:
A small left-over from originally adding the tests here? (I think you can undo this)

Reply:
```suggestion
@pytest.fixture
```
Drop unnecessary () from pytest.fixture